SPECS/python27-alt.spec

```diff
@@ -120,7 +120,11 @@ echo 'install_dir='"%{buildroot}%{bindir}" >> setup.cfg
 # Create lib-dynload directory, then altinstall into %{buildroot} at
 # the ${prefix} (e.g., /usr/local)
 mkdir -p %{buildroot}%{prefix}/lib/python%{pybasever}/lib-dynload
-make altinstall DESTDIR=%{buildroot} PREFIX=%{prefix}
+
+# The -i option ignores all errors in recipes executed to remake files.
+# run this if you have previous python installs impeding your rpm build progress.
+make -i altinstall DESTDIR=%{buildroot} PREFIX=%{prefix}
+#make altinstall DESTDIR=%{buildroot} PREFIX=%{prefix}
```

Comment:
Please reverse these so the line with '-i' is commented out, rather than the line without it.  This way the default is to have error checking on, but it is clear to users how to fix the problem we encountered before by switching which line is being used.